code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -325,83 +326,27 @@
 	"Green" = "#008000", \
 	"Blue" = "#0000FF")
 
-	var/tech_list = list("Rose" = "#F62C6B", \
-	"Peony" = "#E54750", \
-	"Lily" = "#F6562C", \
-	"Daisy" = "#ECCD39", \
-	"Zinnia" = "#89F62C", \
-	"Ivy" = "#5DF62C", \
-	"Iris" = "#2CF6B8", \
-	"Petunia" = "#51A9D4", \
-	"Violet" = "#8A347C", \
-	"Lilac" = "#C7A0F6", \
-	"Orchid" = "#F62CF5")
+	var/color = pick(magic_list)
+	G.name_color = magic_list[color]
+	var/picked_name = pick("Aries", "Leo", "Sagittarius", "Taurus", "Virgo", "Capricorn", "Gemini", "Libra", "Aquarius", "Cancer", "Scorpio", "Pisces")
 
-	var/bio_list = list("Rose" = "#F62C6B", \
-	"Peony" = "#E54750", \
-	"Lily" = "#F6562C", \
-	"Daisy" = "#ECCD39", \
-	"Zinnia" = "#89F62C", \
-	"Ivy" = "#5DF62C", \
-	"Iris" = "#2CF6B8", \
-	"Petunia" = "#51A9D4", \
-	"Violet" = "#8A347C", \
-	"Lilac" = "#C7A0F6", \
-	"Orchid" = "#F62CF5")
+	G.name = "[picked_name] [color]"
+	G.real_name = "[picked_name] [color]"
+	G.icon_living = "[theme][color]"
+	G.icon_state = "[theme][color]"
+	G.icon_dead = "[theme][color]"
 
-	var/picked_name
-//	var/picked_color = pick("#FFFFFF","#000000","#808080","#A52A2A","#FF0000","#8B0000","#DC143C","#FFA500","#FFFF00","#008000","#00FF00","#006400","#00FFFF","#0000FF","#000080","#008080","#800080","#4B0082")
-
-	switch(theme)
-		if("magic")
-			color = pick(magic_list)
-			G.name_color = magic_list[color]
-			picked_name = pick("Aries", "Leo", "Sagittarius", "Taurus", "Virgo", "Capricorn", "Gemini", "Libra", "Aquarius", "Cancer", "Scorpio", "Pisces")
-
-			G.name = "[picked_name] [color]"
-			G.real_name = "[picked_name] [color]"
-			G.icon_living = "[theme][color]"
-			G.icon_state = "[theme][color]"
-			G.icon_dead = "[theme][color]"
-
-			to_chat(user, "[G.magic_fluff_string].")
-		if("tech")
-			color = pick(tech_list) //technically not colors, just flowers that can be specific colors
-			G.name_color = tech_list[color]
-			picked_name = pick("Gallium", "Indium", "Thallium", "Bismuth", "Aluminium", "Mercury", "Iron", "Silver", "Zinc", "Titanium", "Chromium", "Nickel", "Platinum", "Tellurium", "Palladium", "Rhodium", "Cobalt", "Osmium", "Tungsten", "Iridium")
-
-			G.name = "[picked_name] [color]"
-			G.real_name = "[picked_name] [color]"
-			G.icon_living = "[theme][color]"
-			G.icon_state = "[theme][color]"
-			G.icon_dead = "[theme][color]"
-
-			to_chat(user, "[G.tech_fluff_string].")
-			G.speak_emote = list("states")
-		if("bio")
-			color = pick(bio_list) //technically not colors, just using the same flowers as tech currerntly
-			G.name_color = tech_list[color]
-			picked_name = pick("brood", "hive", "nest")
-			to_chat(user, "[G.bio_fluff_string].")
-
-			G.name = "[color] [picked_name]"
-			G.real_name = "[color] [picked_name]"
-			G.icon_living = "[theme][color]"
-			G.icon_state = "[theme][color]"
-			G.icon_dead = "[theme][color]"
-
-			to_chat(user, "[G.bio_fluff_string].")
-			G.attacktext = "swarms"
-			G.speak_emote = list("chitters")
+	to_chat(user, "[G.magic_fluff_string].")
```

Comment:
is this debug text 👀

Reply:
Nah the guardian has their own version depending on the type

Reply:
Oh its telling them what they got. Right, okay.